mapswipe_workers/mapswipe_workers/project_types/media_classification/project.py

```diff
@@ -35,9 +35,18 @@ def __init__(self, project_draft: dict):
         self.mediaCredits = project_draft.get("mediaCredits", None)
         self.medialist = []
         self.mediaurl = project_draft["mediaurl"]
-        self.answerLabels = project_draft.get("answerLabels", None)
         self.get_media()
 
+        self.answerLabels = project_draft.get(
+            "answerLabels",
+            [
+                {"color": "", "label": "no", "value": 0},
+                {"color": "green", "label": "yes", "value": 1},
+                {"color": "orange", "label": "maybe", "value": 2},
+                {"color": "red", "label": "bad imagery", "value": 3},
+            ],
```

Comment:
So media projects have no "bad data" answer possibility? Do you think adding one could make sense, or is the input data supposed to be screened beforehand so no bad data gets mixed in?

Reply:
In my opinion, "bad data" would not necessarily be among the **default** options as the images/videos in media projects will often be carefully curated. It could become more important once we start creating projects based on e.g. randomly selected street level imagery, but could be added as a custom option by project creators in those cases anyway.